oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -124,30 +166,37 @@ volunteers are needed, we will put out a call to join. Volunteers will need to a
 and will then be evaluated by the existing working group members. We hope to have six to seven members of the working
 group at a time.
 
-Security work will be a mix of private and public tickets
----------------------------------------------------------
-Proactive work for security improvements will be made public, while vulnerability reports will be handled privately.
-We propose using Github Security Advisories to handle the triage of vulnerability reports to the owners of
-vulnerable components.
+There will also be ways to participate in the security of the Open edX platform without being a member of the working group. Maintainers and core contributors of the Open edX community interested in volunteering to implement security hotfixes for other maintainers who may not have the bandwidth to immediately address vulnerabilities are encouraged to email their interest to security@openedx.org.
+
+Visibility
+----------
+Security work will be a mix of private and public tickets. Proactive work for security improvements will be made public, while vulnerability reports will be handled privately.
 
-Security Team Member Responsibilities
--------------------------------------
+We propose using GitHub Security Advisories to handle the triage of vulnerability reports to the owners of vulnerable components.
```

Comment:
Not a problem to note here but I'm realizing that we'll need to keep a master list of draft advisories as a working group because github doesn't really provide a way to do that yet at the organization level.